src/backend/api/meals.js

```diff
@@ -86,4 +142,26 @@ router.delete('/:id', async (req, res) => {
   }
 });
 
+router.get('/:meal_id/reviews', async (req, res) => {
+  const mealId = req.params.meal_id;
+  const query = knex
+    .select('review.*')
+    .from('review')
+    .join('meal', 'review.meal_id', '=', `meal.id`)
+    .having('meal_id', '=', mealId);
+
+  console.log('SQL', query.toSQL().sql);
+
+  try {
+    const reviews = await query;
+    if (reviews.length !== 0) {
+      res.json(reviews);
+    } else {
+      res.status(404).json({ error: 'No reviews found' });
```

Comment:
Same comment as before: I would not recommend a 404 response here, returning just an empty list of reviews is better IMO.